### PR TITLE
LPS-74526 Create npm scripts and apply linting and formatting guidelines

### PR DESCRIPTION
**Pull request by jbalsas**

### CODE REVIEW

**Comment by mthadley** [comment]
@jbalsas In version [`1.6.0`](https://github.com/prettier/prettier/releases/tag/1.6.0) of `prettier`, they added support for a configuration file, in case you want to move it out of the npm script (like `eslint`).

It also makes it easier for developers who might be running `prettier` on save in their editor, since it should pick up the configuration from the project.

**Comment by Robert-Frampton** [comment]
Hey @jbalsas looks good to me.

File: modules/apps/foundation/frontend-js/frontend-js-web/.eslintrc [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	},
+    "extends": "liferay",
+    "globals": {
+        "Liferay": true
+    },
+	"rules": {
+        "camelcase": "off",
+        "object-curly-spacing": "off"
+    }
+}
```
Comment (Robert-Frampton) [inline modules/apps/foundation/frontend-js/frontend-js-web/.eslintrc]: Hey @jbalsas, you probably noticed already, but this file has mixed spaces/tabs.

Comment (jbalsas) [reply modules/apps/foundation/frontend-js/frontend-js-web/.eslintrc]: I actually didn't 🙈

**Comment by jbalsas** [comment]
Hey @mthadley, thanks for pointing that out! I went scanning through their docs and found no reference to it having been implemented, so I thought they hadn't released that yet. I'll make sure to update that and move the config to a configuration file. This should also make it easier to reuse across projects and document in the guidelines! 🎉

**Comment by jbalsas** [comment]
Sent to brian: https://github.com/brianchandotcom/liferay-portal/pull/51398

File: modules/apps/foundation/frontend-js/frontend-js-web/.eslintrc (hunk 1 above)
Comment (blzaugg) [reply modules/apps/foundation/frontend-js/frontend-js-web/.eslintrc]: If only we had a linter for this.

**Pull request closed without merging**